File: 2019/03/ruby/main.rb
#!/usr/bin/env ruby

require_relative 'probe'

input = File.readlines("../input.txt")
paths = input.map { |line| line.chomp.split(',') }
probe = Probe.new(paths)
p "Closest: #{probe.closest_intersection} | Fewest: #{probe.fewest_steps}"

File: 2019/03/python/main.py
#!/usr/bin/env python3

from probe import Probe

def run():
  f = open('../input.txt', 'r')
  paths = [line.strip().split(',') for line in f.readlines()]
  f.close()
  probe = Probe(paths)
  print("Closest: {} | Fewest: {}".format(probe.closest_intersection(), probe.fewest_steps()))


if __name__ == '__main__':
  run()